docs/release-notes/1376-static-swagger-docs

```diff
@@ -0,0 +1,4 @@
+:orphan:
+
+**New Features**
+- Bundled a version of Swagger-ui to Determiend documentation.
```

Comment:
maybe 
"Determined API documentation (Swagger-UI) is now bundled with the rest of docs and available at docs.determined.ai". ?